frontend/components/tile/network-capacity-chart.html

```diff
@@ -89,47 +112,48 @@
         window.addEventListener('WebComponentsReady', onComponentsReady);
 
         function onComponentsReady() {
-          $chart.width = component.clientWidth * 0.96;
-          $chart.height = document.querySelector('.app-logo').clientHeight * 0.96;
+          $chart.width  = component.clientWidth;
+          $chart.height = document.querySelector('.app-logo').clientHeight;
+          $chart.style.width = component.clientWidth  + 'px';
+          $chart.style.height = document.querySelector('.app-logo').clientHeight + 'px';
+        }
+      },
+      smootheChartFactory: function (options) {
+        if (!this._chart) {
+          var msPerPixel = (1000 * 60 * (options.maxCharPoints - 1)) / this.$.myChart.width;
+          var chart = new SmoothieChart({
+            millisPerPixel: msPerPixel,
+            labels:{
+              precision: 0,
+              fontSize: 14,
+            },
+            grid: {
+              strokeStyle: '#555555',
+              lineWidth: 1,
+              millisPerLine: 60000,
+              verticalSections: 6
+            },
+            timestampFormatter: function (date) {
+              if (date.getMinutes() % 2 === 1) {
+                return '';
+              }
+              return date.toISOString().slice(11, 16);
```

Comment:
You will get bad time since toISOString() returns in UTC timezone. I would suggest `date.getHours() + ':' + date.getMinutes()` instead.